src/services/fileStorage/utils/filePathHelper.js

```diff
@@ -1,9 +1,32 @@
 const removeLeadingSlash = (path) => (path[0] === '/' ? path.substring(1) : path);
 
-const generateFileNameSuffix = (fileName) => {
+const s3FileNameFilterExpression = /[^0-9A-Za-z\-_.]+/g;
+/**
+ * ensures a valid filename for s3 removing invalid characters from local/internal filename
```

Comment:
The description can be confused because it is also for our collection, or more generally naming cleanup.

Reply:
suggestion?